gcinvoice.py

```diff
@@ -297,6 +297,8 @@ def parse(self, gcfile=None):
                 invcdict['notes'] = invc.findtext(ns('invoice:notes'))
                 invcdict['currency'] = invc.findtext(
                         ns('invoice:currency/cmdty:id'))
+
+                invcdict['owner']['id'] = str(invcdict['owner']['id']).zfill(5)
```

Comment:
You forgot to change in test.py to change (two occurences on lines 208 and 213) from
'id': 1 -> 'id': '00001',